jOOQ-examples/jOOQ-spring-boot-example/src/test/java/org/jooq/example/spring/QueryTest.java

```diff
@@ -19,15 +19,15 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.SpringApplicationConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
 
 /**
  * @author Lukas Eder
  * @author Thomas Darimont
  */
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = Application.class)
+@RunWith(SpringRunner.class)
```

Comment:
The other runner is just icing on the cake atm, the SpringBootTest autodetects the "main" configuration class if it's in the same package, usually people omit if not necessary.